### DISCUSSION TITLE
NPC 걷기 설정

### DISCUSSION THREAD

**Discussion by ryusiin** [naverz/zepeto-studio-kor#507] [5. Q&A]
목적 : NPC가 이동할 때 뛰지 못하고 계속 걷게 하고 싶습니다.

알고 싶은 것 : 
애니메이션 적용이 아닌, ZepetoCharacter를 조작해서 걷기만 하도록 제한하는 법이 있을까요?

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요.
제페토 캐릭터를 생성하는 방식이 2가지가 있는데,
1. https://studio.zepeto.me/guides/zepetoplayer
2. https://studio.zepeto.me/guides/zepetocharacter

1번의 경우 제페토캐릭터모델과 제어기능까지 포함한 생성이고
2번의 경우 단순한 제페토캐릭터모델 생성을 위한 방법을 제공하고있습니다.

NPC의 경우 2번 방식으로 생성을 하면 좋을 것 같고
질문해주신 내용이 기존 제어기능에서 걷기만 가능하게 커스텀이 가능하냐는 부분은 현재로는 제공이 불가능해서
기존 제어기능을 비활성화하고 새롭게 만들어서 붙여주셔야 합니다.
